src/test/pybind/test_ceph_argparse.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+# use the script to setup the nosetest environment as it should be.
+# Mainly due /usr/bin/env uses a parameter setup different from what
+# python expects
+
+[ ! -z $CEPH_ROOT ] || exit 1
+
+PYBIND=$CEPH_ROOT/src/pybind /usr/local/bin/nosetests  --stop $CEPH_ROOT/src/test/pybind/test_ceph_argparse.py
```

Comment:
@wjwithagen I agree with @tchaikov, it would be better to just use **#!/usr/bin/env nosetest** to resolve that issue. It is simpler and looks like it would work for FreeBSD as well.